### DISCUSSION TITLE
Where can I donate?

### DISCUSSION THREAD

**Discussion by jamesah2008** [wiredopposite/OGX-Mini#18] [Q&A]
I'm really enjoying this software and appreciate your work. Where can I make a small donation? I can send to you or an organization you'd like to support.

**Comment by wiredopposite** [comment]
> I'm really enjoying this software and appreciate your work. Where can I make a small donation? I can send to you or an organization you'd like to support.

Hey I appreciate that, I setup a Ko-Fi page where people can help fund more controllers: https://ko-fi.com/wiredopposite If you'd like to see a specific controller added, just make sure to include a note with the donation and maybe a link to where it's sold.

If you'd like to support and want something in return, I do also run a webstore at [wiredopposite.com](wiredopposite.com) with mostly Xbox related things for sale.
